README.md

```diff
@@ -375,6 +375,17 @@ transloco-keys-manager find --translations-path my/path
 transloco-keys-manager find -p my/path
 ```
 
+- `getTextMarker`: The marker sign for dynamic values only for extract key: (defaults to `marker`)
```

Comment:
This shouldn't be an option, we should resolve the marker name ourselves.

Reply:
it is needed for support alias if marker conflicted with users functions
see example: https://github.com/biesbjerg/ngx-translate-extract-marker/tree/master/projects/ngx-translate-extract-marker

Reply:
@EndyKaufman just because that's how they implement it doesn't mean that's how we want to implement it 🙂 
Given the following import statement:
```typescript
import { marker as something } from '@ngneat/transloco-keys-manager'
```
I don't need the user to specify what's the marker, I can extract it.

Reply:
done

Reply:
done